backend/app/common/jwt.py

```diff
@@ -42,28 +43,34 @@ def password_verify(plain_password: str, hashed_password: str) -> bool:
     return pwd_context.verify(plain_password, hashed_password)
 
 
-def create_access_token(data: int | Any, expires_delta: timedelta | None = None) -> str:
+async def create_access_token(sub: int | Any, data: dict, expires_delta: timedelta | None = None) -> str:
     """
     Generate encryption token
 
+    :param sub: The subject/userid of the JWT
     :param data: Data transferred to the token
     :param expires_delta: Increased expiry time
     :return:
     """
     if expires_delta:
         expires = datetime.utcnow() + expires_delta
+        expire_seconds = expires_delta.total_seconds()
     else:
         expires = datetime.utcnow() + timedelta(settings.TOKEN_EXPIRE_MINUTES)
```

Comment:
Hi, @downdawn 

We have a bug here that needs to be fixed, the parameter in timedelta needs to be specified minutes

Reply:
timedelta(minutes=settings.TOKEN_EXPIRE_MINUTES)

Reply:
Or we can change it to be timed in seconds

Reply:
```
TOKEN_EXPIRE_SECONDS: int = 60 * 24 * 1 * 60  # token 时效 60 * 24 * 1 * 60 = 1 天
```
```
    if expires_delta:
        expires = datetime.utcnow() + expires_delta
        expire_seconds = expires_delta.total_seconds()
    else:
        expires = datetime.utcnow() + timedelta(seconds=settings.TOKEN_EXPIRE_MINUTES)
        expire_seconds = settings.TOKEN_EXPIRE_SECONDS
```

Reply:
Great!

Reply:
WIP